### DISCUSSION TITLE
GFN0-xTB under PBC

### DISCUSSION THREAD

**Discussion by lcdamoy** [grimme-lab/xtb#529] [Q&A]
Hi, all, i used GFN0 and GFN1 to optimize the molecular crystal unders PBC with xtb code, i found that GFN0 will optimize the lattice constants in the meantime, while GFN1 will not. Both in the same configuration. Is there some measure to fix lattice constants?

**Comment by awvwgk** [comment]
The PBC implementation in `xtb` is somewhat unfinished at the moment. I'm currently integrating the xTB methods in the DFTB+ program package to allow using their periodic framework. With the DFTB+ 21.2 release the xTB Hamiltonian is now supported via the tblite library.

**Reply by zhangyue1723** [reply]
I use cp2k to perform xtb calculation the reason is also the changing of lattice parameter. But another question is, I should use the Tag of "Shake" to control the dissociation of H atom, but there seems no same control method in cp2k?
